apiary-metastore-events/README.md

```diff
@@ -4,9 +4,10 @@
 Apiary Metastore Events contains a set of modules responsible for retrieving and processing Hive Metastore events.
 
 Currently, the following modules are defined:
- - [apiary-metastore-listener](apiary-metastore-listener) - listens to events from the Hive Metastore and publishes them to an SNS queue.
- - [apiary-receivers](apiary-receivers) - provides an implementation for receiving and deserializing Hive events from a queue.
- 
+ - [apiary-metastore-listener](apiary-metastore-listener) - listens to events from the Hive Metastore and publishes them to an SNS queue
+ - [apiary-receivers](apiary-receivers) - common logic to simplify receiving and deserialising Hive Metastore event messages published by the 'apiary-metastore-listener'
+ - [apiary-metastore-consumer](apiary-metastore-consumer) - consumers that receive and process Hive Metastore event messages published by the 'apiary-metastore-listener'
```

Comment:
I think we should rename this to `apiary-metastore-consumers` (i.e. plural) to be in line with the convention in the rest of this project where we have plurals for the top-level modules which may have mutliple sub-module implementations (e.g. `apiary-receivers`). I'll do this now.